app/controllers/spree/api/v2/storefront/confirmations_controller.rb

```diff
@@ -0,0 +1,20 @@
+module Spree
+  module Api
+    module V2
+      module Storefront
+        class ConfirmationsController < ::Spree::Api::V2::BaseController
+
+          def show
+            user = Spree.user_class.confirm_by_token(params[:confirmation_token])
```

Comment:
@pmilewski will it return a 404 when not found?

Reply:
422: Confirmation token is invalid